### PR TITLE
Add welcome screen for portfolio

### LINKED ISSUE #12: Footer move to bottom of screen
Move footer to the bottom of screen on home page and slides page.

Screenshots:
https://screenshot.googleplex.com/utw32RbJ4OF.png
https://8080-5efc772e-925a-4fef-a0ac-77c0f89b1684.us-west1.cloudshell.dev/?authuser=0

### PR DESCRIPTION
**Pull request by jaysaleh**
I've added a home screen to my portfolio which includes a header, body with and image and short description of myself, and a footer containing social media links. Tools used include HTML and CSS.

Screenshots:
https://screenshot.googleplex.com/pgNCCUdG92L

### CODE REVIEW

**Review by billie-cat — APPROVED** [review]
Please wait for Fola to approve

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,91 @@
...
+    flex-direction: row;
+    list-style:none;
+}
+
+#header a{
+    text-decoration: none;
+    color: #000;
+    padding-right: 35px;
+    font-size: 18px;
+    font-weight: 200;
```
Comment (f-camp) [inline portfolio/src/main/webapp/style.css]: In the spirit of following the style guide we should alphabetize declarations.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Please do this in the other declarations in this file too.

Comment (jaysaleh) [reply portfolio/src/main/webapp/style.css]: Made fixes including, alphabetization, reducing indentations to 2 spaces, making hexadecimal color codes lowercase, adding semicolons to every line, and including multimedia fallbacks.

**Review by f-camp — APPROVED** [review]

**Comment by billie-cat** [comment]
Does this need to be merged?

**Comment by jaysaleh** [comment]
Yes, planning on merging everything at once.

**Comment by jaysaleh** [comment]
#12

**Comment by billie-cat** [comment]
LGTM, though can you update the title of this pull request. You want to make it clear what changed. The current title isn't clear what changed. Consider this simple change: "Add welcome screen for portfolio"

**Pull request merged**